models/reviews.model.js

```diff
@@ -92,17 +94,36 @@ exports.insertComment = (id, comment) => {
   const { review_id } = id;
   const { username, body } = comment;
 
-  const queryStr = `INSERT INTO comments(author, body, review_id)
+  const queryStr = `
+  INSERT INTO comments(author, body, review_id)
   VALUES ($1, $2, $3)
-  RETURNING*;`;
+  RETURNING*;
+  `;
 
   const values = [username, body, review_id];
 
   return db.query(queryStr, values).then((result) => {
     if (result.rows.length === 0) {
       return checkExists("users", "username", username);
     }
+    return result.rows[0];
+  });
+};
+
+exports.updateVotesById = (id, votes) => {
+  const { inc_votes } = votes;
 
+  const queryStr = `
+  UPDATE reviews 
+  SET votes = votes + $1::INT
+  WHERE review_id = $2
+  RETURNING*;
+  `;
+
+  return db.query(queryStr, [inc_votes, id]).then((result) => {
+    if (result.rows.length === 0) {
+      return checkExists("reviews", "review_id", id);
+    }
```

Comment:
In the event that the length of the rows is zero, I think that is confirmation that the review doesn't exist! No need for another function call, you can go ahead and `Promise.reject()` quite happily 👍